ablator/analysis/main.py

```diff
@@ -53,28 +53,29 @@ def _parse_results(
 
 class Analysis:
     """
-    A class for analyzing experimental results.
+    A class that stores and processes the attributes, metrics, and other data for the plotting
+    of the experiment result.
 
     Parameters
     ----------
     results : pd.DataFrame | Results
         The result dataframe.
     categorical_attributes : list[str] | None
-        The list of all the categorical hyperparameter names
+        The list of all the categorical hyperparameter names, by default ``None``.
     numerical_attributes : list[str] | None
-        The list of all the numerical hyperparameter names
+        The list of all the numerical hyperparameter names, by default ``None``.
     optim_metrics : dict[str, Optim] | None
-        A dictionary mapping metric names to optimization directions.
+        A dictionary mapping metric names to optimization directions, by default ``None``.
     save_dir : str | None
-        The directory to save analysis results to.
+        The directory to save analysis results to, by default ``None``.
     cache : bool
-        Whether to cache results.
+        Whether to cache results, by default ``False``.
```

Comment:
I have seen it both as ``False`` and False. What is standardized?

Reply:
I actually did not check if there's a standard to document default values and if we should use `False` or False. Will double check!

Reply:
According to this in numpydoc style guide: https://numpydoc.readthedocs.io/en/latest/format.html#:~:text=%60x%60%20(-,the%20default%20is%20%2D1,-%2C%20which%20implies%20summation it seems like it's quite flexible. Also, I used `False` instead of False as a way to make its look stand out instead of just normal text